### PR TITLE
LTD-5276: Add field to save products ITAR status

### PR DESCRIPTION
**Pull request by saruniitr**
## Change description

We are adding a new question in the security approvals flow to ask if the any of the products are subject to ITAR controls.
This is a yes/no question so add a new boolean field to save answer.

If we create a new application by copying an existing one (amendment) then this field also gets copied across.

Add tests to ensure this is correctly created and updated as expected.

Related FE PR https://github.com/uktrade/lite-frontend/pull/2103

[LTD-5276](https://uktrade.atlassian.net/browse/LTD-5276)

[LTD-5276]: https://uktrade.atlassian.net/browse/LTD-5276?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api/applications/models.py [hunk 1]
```diff
@@ -320,6 +320,7 @@ class StandardApplication(BaseApplication, Clonable):
     f1686_reference_number = models.CharField(default=None, blank=True, null=True, max_length=100)
     f1686_approval_date = models.DateField(blank=False, null=True)
     other_security_approval_details = models.TextField(default=None, blank=True, null=True)
+    subject_to_itar_controls = models.BooleanField(blank=True, default=None, null=True)
```
Comment (depsiatwal) [inline api/applications/models.py:323]: is this mean't to be boolean ?

Comment (saruniitr) [reply api/applications/models.py:323]: yes, it is a yes/no question

Comment (saruniitr) [reply api/applications/models.py:323]: The form is in the linked FE PR https://github.com/uktrade/lite-frontend/pull/2103

**Review by depsiatwal — APPROVED** [review]
lgtm

**Review by hnryjmes — APPROVED** [review]
happy with code changes

**Pull request merged**